### ISSUE TITLE
Why are admin rights required to load a jar in windows?

### ISSUE THREAD

**Issue by TBRC-JimK** [buda-base/asset-manager#33]

**Comment by TBRC-JimK** [comment]
I could try just putting the test lib on the classpath.

**Comment by TBRC-JimK** [comment]
On NT's machine. NT has to run it from the program installs directory, with the security policy. If you move away, and run it form another working dir, you still get the ClassNotFound exception
See [ClassNotFoundException issued by URLClassLoader when the Security Manager is enabled](https://stackoverflow.com/questions/57433655/classnotfoundexception-issued-by-urlclassloader-when-the-security-manager-is-ena)

**Comment by TBRC-JimK** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [NT install and run](https://trello.com/c/7aStbfOO/60-nt-install-and-run)

**Comment by TBRC-JimK** [comment]
So, I have a VMBox setup, and I just downloaded "Java" and got the 32 bit version. I also created an ordinary user joschmidt to test this in. I put the install in

**Comment by TBRC-JimK** [comment]
on my first try, I took my new user and downloaded at9 from a share. I got the expected `...script cannot be loaded because running scripts..... See https://go.microsoft.com/fwlink/?LinkID=135710`

Then I remembered I downloaded the stuff from a share, and what I need to do is download it from "the internet" . Easy enough to access RS3:/home from inner.tbrc.org:6102

[ ] Ask NT: what does Get-ExecutionPolicy -List show?

OK, with the 32 bit Java, doesn't fail, period.
